plugins/system/shortcut/shortcut.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ * @package     Joomla.Plugins
+ * @subpackage  System.shortcut
+ *
+ * @copyright   (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Application\AdministratorApplication;
+use Joomla\CMS\Component\ComponentHelper;
+use Joomla\CMS\Event\GenericEvent;
+use Joomla\CMS\Factory;
+use Joomla\CMS\Plugin\CMSPlugin;
+use Joomla\CMS\Plugin\PluginHelper;
+use Joomla\Event\Event;
+use Joomla\Event\SubscriberInterface;
+
+/**
+ * Shortcut plugin to add accessible keyboard navigation to the site and administrator templates.
```

Comment:
```suggestion
 * Shortcut plugin to add accessible keyboard shortcuts to the site and administrator templates.
```

Reply:
Good that you saw that. I removed the part about the frontend